src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -49,29 +51,17 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         this.eventNames = eventNames;
         this.options = $.extend({}, defaults, options);
         overlaysControl = new CAC.Map.OverlaysControl();
-        map = new cartodb.L.map(this.options.id, { zoomControl: false })
-                           .setView(this.options.center, this.options.zoom);
 
-        tabControl = options.tabControl;
-        homepage = options.homepage;
+        this.itineraryControl = new CAC.Map.ItineraryControl({map: null});
 
-        // put zoom control on top right
-        zoomControl = new cartodb.L.Control.Zoom({ position: 'topright' });
-
-        initializeBasemaps();
-
-        if (!homepage) {
-            // hide zoom control on home page view
-            zoomControl.addTo(map);
-            // delay loading overlays and layer switcher
-            initializeOverlays();
-            initializeLayerControl();
+        // only load map tiles if map visible
+        if (!homepage || $(this.options.selector).is(':visible')) {
```

Comment:
This seemed the most resilient option to me, as it's independent of where the page width breaks happen. If the map selector changes, it would be pretty obvious that we'd need to update that selector in our code.